src/Components/NavBar/Menus/MobileNavMenu.tsx

```diff
@@ -82,15 +82,39 @@ const MobileLink: React.FC<MobileLinkProps> = ({
   children,
   ...props
 }) => {
+  const { sm: isTablet } = useMedia()
+  const [isPressed, setPressed] = useState(false)
+  const bg = isTablet && isPressed ? "black5" : "white100"
+  const getEvents = () => {
```

Comment:
#minor but this probably doesn't need to be deferred? As in, doesn't need to be a function.

Reply:
It's for spreading in; it needs to return an empty object if a condition isn't met.